views/layout.ejs

```diff
@@ -6,7 +6,8 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <title><%= title %> - <%= process.env.APP_NAME %></title>
-    <link rel="stylesheet" href="/libraries/bootstrap/css/bootstrap.css" />
+      <!-- Bootstrap core CSS -->
+    <link href="stylesheets/bootstrap.css" rel="stylesheet">
```

Comment:
Did it not work to simply override certain styles in Bootstrap?

By copying the entire file and then modifying specific styles, that makes it more difficult to upgrade the Bootstrap version...?